docs/guide/adding-custom-widget-placeholder-content.md

```diff
@@ -0,0 +1,126 @@
+
+Next, to supply the URL of the placeholder content, the module should invoke the `determineBestAssetUrl('placeholder')` method in the `init(self)` initialization function.
+
+Even though the option name ends with `Image`, this same method can be used for adding custom video placeholder content that will be uploaded into the module's `public` folder. In contrast, for videos that are hosted externally, you need to add the `placeholder: true` and `placeholderUrl` option with the value set to the URL of the resource. The initialization function in this case can be skipped.
+
+Example for adding a custom image.
+
+<AposCodeBlock>
+
+```javascript
+module.exports = {
+  extend: '@apostrophecms/widget-type',
+  options: {
+    label: 'Custom Widget',
+    placeholder: true,
+    placeholderImage: 'png'
+  },
+  init(self) {
+    self.determineBestAssetUrl('placeholder');
+  },
+  // remainder of the widget code
+};
+```
+
+<template v-slot:caption>
+  modules/custom-widget/index.js
+</template>
+
+</AposCodeBlock>
+
+::: note
+You can bypass the call to `determineBestAsetUrl()` for your images or videos stored in the `public` folder and instead pass the path to the asset using `placeholderUrl` in place of `placeholderImage`. This is discouraged because errors can be made in this path. It is better to let Apostrophe figure this out for you.
+:::
+
+### Altering the widget.html file
+
+The specific alteration of the `widget.html` template will depend on the type of placeholder content being delivered. Irrespective of file type, the template will have access to `data.widget.aposPlaceholder` and `data.manager.options.placeholderUrl` to populate the markup. 
+
+The first, `data.widget.aposPlaceholder`, allows for confirmation that the placeholder should be displayed. This will return `true` when the widget is first added and return `false` once the widget has been edited.
+
+The second, `data.manager.options.placeholderUrl`, will contain the path to the content placeholder asset. This will either be the path computed by the `determineBestAssetUrl()`, or the URL passed directly through the `placeholderUrl` option.
+
+This example demonstrates adding an image.
+
+<AposCodeBlock>
+
+```twig
+<section data-custom-widget>
+  <h1>Custom Widget</h1>
+  {% if data.widget.aposPlaceholder and data.manager.options.placeholderUrl %}
+  <img
+    src="{{ data.manager.options.placeholderUrl }}"
+    alt="{{ __t('nameSpace:imagePlaceholder') }}"
+    class="custom-widget-placeholder"
+  />
+  {% endif %}
+</section>
+```
+<template v-slot:caption>
+  modules/custom-widget/views/widget.html
+</template>
+
+</AposCodeBlock>
+
+This example demonstrates adding a video. Depending on where the video is delivered from, you can either take advantage of the `@apostrophecms/oembed` and `@apostrophecms/video-widget` module's abilities to handle these types of content automatically. For project-uploaded videos, you will need to create appropriate markup.
```

Comment:
Your example here is for a self-hosted video that's a static asset. Let's just stick to that. I think the discussion of using the oembed module or video-widget is confusing.

Reply:
Changed.